huaweicloud/resource_huaweicloud_gaussdb_mysql_instance.go

```diff
@@ -294,6 +298,8 @@ func resourceGaussDBInstanceCreate(d *schema.ResourceData, meta interface{}) err
 		}
 	}
 
+	lowerCaseTableNames := d.Get("lower_case_table_names").(int)
```

Comment:
the default value will be **0** while it defaults to **1** in the docs, please check this.

Reply:
fixed